common/src/main/java/muramasa/gregtech/loader/machines/ChemicalReactorLoader.java

```diff
@@ -6,58 +6,180 @@
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(Carbon,1)),INT_CIRCUITS.get(2).setNoConsume()).fi(Oxygen.getGas(144)).fo(CarbonDioxide.getGas(144)).add(80, 8);
+        CHEMICAL_REACTING.RB().fi(NitricAcid.getLiquid(4*144),MethylAcetate.getLiquid(288)).io(DUST.get(Carbon,5)).fo(Tetranitromethane.getLiquid(144),Water.getLiquid(8*144)).add(480, 120);
+        CHEMICAL_REACTING.RB().ii(INT_CIRCUITS.get(1).setNoConsume()).fi(Oxygen.getGas(7*144),Ammonia.getGas(4*144)).fo(DinitrogenTetroxide.getGas(288),Water.getLiquid(6*144)).add(480, 30);
+        CHEMICAL_REACTING.RB().ii(INT_CIRCUITS.get(2).setNoConsume()).fi(Oxygen.getGas(288),Ammonia.getGas(144)).fo(NitricAcid.getLiquid(144),Water.getLiquid(144)).add(480, 30);
+        CHEMICAL_REACTING.RB().ii(INT_CIRCUITS.get(2).setNoConsume()).fi(NitrogenDioxide.getGas(288)).fo(DinitrogenTetroxide.getGas(144)).add(640, 30);
+        CHEMICAL_REACTING.RB().fi(Methanol.getLiquid(144),SeedOil.getLiquid(6*144)).fo(Glycerol.getLiquid(144),BioDiesel.getLiquid(6*144)).add(600, 30);
+        CHEMICAL_REACTING.RB().fi(Ethanol.getLiquid(144),SeedOil.getLiquid(6*144)).fo(Glycerol.getLiquid(144),BioDiesel.getLiquid(6*144)).add(600, 30);
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(Carbon,2)),of(DUST.get(Calcium,2))).fi(Oxygen.getGas(3*144)).io(DUST.get(Calcite,2)).add(500, 30);
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(Quicklime,1))).fi(CarbonDioxide.getGas(144)).io(DUST.get(Calcite,1)).add(80, 30);
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(Magnesia,1))).fi(CarbonDioxide.getGas(144)).io(DUST.get(Magnesite,1)).add(80, 30);
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(SodiumHydroxide,4))).fi(Dichlorobenzene.getLiquid(288)).io(DUST.get(Salt,4)).fo(Phenol.getLiquid(288),Oxygen.getGas(144)).add(120, 30);
+    }
+
+    private static void alkaloids_2_alkanoles(){
+        alkane_2_alkanole(Methane,Methanol,2,1);
+        alkane_2_alkanole(Ethane,Ethanol,2,1,INT_CIRCUITS.get(1).setNoConsume());
+        alkane_2_alkanole(Ethane,Ethanediol,1,1,INT_CIRCUITS.get(2).setNoConsume());
+        alkane_2_alkanole(Propane,Propanol,2,1,INT_CIRCUITS.get(1).setNoConsume());
+        alkane_2_alkanole(Propane,Propanediol,1,1,INT_CIRCUITS.get(2).setNoConsume());
+        alkane_2_alkanole(Propene,Propenol,2,1);
+        alkane_2_alkanole(Butane,Butanol,2,1,INT_CIRCUITS.get(1).setNoConsume());
+        alkane_2_alkanole(Butane,Butanediol,1,1,INT_CIRCUITS.get(2).setNoConsume());
+        alkane_2_alkanole(Butene,Butenol,2,1);
+    }
+
+    private static void alkanoles_2_alkenes(){
+        alkanole_2_alkene(Ethanol,Ethylene,1,1);
+        alkanole_2_alkene(Propanol,Propene,1,1);
+        alkanole_2_alkene(Butane,Butene,1,1);
+        alkanole_2_alkene(Butanediol,Butadiene,1,2);
+    }
+
+    private static void alkane_2_alkanole(Material alkane, Material alkanole, int mol_alk, int mol_ox, RecipeIngredient... circuit){
```

Comment:
Rename this and any other methods that don't use camel case to camel case please

Reply:
This should be sorted now